lmdb/mdb.c

```diff
@@ -2103,7 +2103,7 @@ static txnid_t
 mdb_find_oldest(MDB_txn *txn)
 {
 	int i;
-	txnid_t mr, oldest = txn->mt_txnid - 1;
+	txnid_t mr, oldest = txn->mt_txnid;
```

Comment:
Maybe I am missing something but this line should not be changed.
The oldest txnId should always be behind the current txnId as, imho, a transaction should not reuse pages it is freeing before a commit.

Reply:
You are right, I have reverted it, and added comment